### ISSUE TITLE
Accessibility of the Oauth page

### ISSUE THREAD

**Issue by bl-aire** [jupyterhub/jupyterhub#4280]
### Details from report:
Based on evaluation using the WAVE accessibility tool, the following are responsible for accessibility issues on the oauth page:

- **Reoccurring accessibility issues**: issues like a very **Low contrast** between nav items and their background, **Language missing or invalid** and **Missing first level header** have been fixed in PR #4274

- **Low contrast between button text and its background**: as discussed, choosing to darken the background will affect JupyterHub's branding as the current colour is part of its brand colours so this is on hold for now.

### Report:

<img width="1280" alt="Screenshot 2022-12-20 at 13 04 04" src="https://user-images.githubusercontent.com/84882370/208912434-fcc2396e-e7a8-4853-bed4-c71d30e574b0.png">

@minrk

**Comment by minrk** [comment]
Great! That sounds like there are no issues for this page?

**Comment by bl-aire** [comment]
> Great! That sounds like there are no issues for this page?

Yes.

**Comment by minrk** [comment]
Thanks for reviewing!
